packages/web-components/src/components/rux-segmented-button/rux-segmented-button.scss

```diff
@@ -1,131 +1,112 @@
 :host {
-    display: block;
-    box-sizing: border-box;
-}
-:host([disabled]) {
-    opacity: var(--disabled-opacity);
-    cursor: not-allowed;
-    pointer-events: none;
+    display: inline-flex;
 
-    label {
-        border-color: var(--color-background-interactive-default);
-    }
+    /* layout */
+    flex-flow: column;
 }
 
 :host([hidden]) {
+  display: none;
+}
+
+/* reset unorganized list */
+ul {
+    list-style-type: none;
+    margin-block: 0;
+    padding-inline: 0;
+}
+
+/* remove input display */
+input {
     display: none;
 }
 
-*,
-*::before,
-*::after {
-    box-sizing: inherit;
+/* reset label style */
+label {
+    cursor: inherit;
 }
 
-.rux-segmented-button {
-    display: inline-flex;
-    // Default is small for 6.0 - will update default to be medium in 7.0
-    height: 1.625rem;
-    padding: 0;
-    margin: 0;
-    list-style: none;
+.rux-segmented-buttons {
+    display: flex;
+
+    /* appearance */
     border-radius: var(--radius-base);
-    background-color: var(--color-background-base-default);
-    &--large {
-        height: 2.875rem;
-    }
-    &--medium {
-        height: 2.125rem;
-    }
 }
 
-.rux-segmented-button__segment {
-    width: auto;
-    margin: 0;
-    padding: 0;
+.rux-segmented-button {
+    /* reference */
+    --border-color: var(--color-text-interactive-default);
```

Comment:
border color should be `color-border-interactive-muted` I believe

NIT/q: could we keep private component custom props on :host at the top still? or does that have unintended side effects? or maybe we just put a comment at the top? It'll be really helpful when we start implementing component tokens to be able to see all the internals at a glance. (2/10)

Reply:
Moved to the top and using `--color-border-interactive-muted`.